v2/tools/scripts/generate.sh

```diff
@@ -4,6 +4,8 @@ set -e
 OPENAPI_FOLDER=${OPENAPI_FOLDER:-"./openapi"}
 OPENAPI_FILE="$OPENAPI_FOLDER/atlas-api.yaml"
 TRANSFORMED_FILE="$OPENAPI_FOLDER/atlas-api-transformed.yaml"
+## Version of the versioned API
+CLIENT_VERSION=${CLIENT_VERSION:-"1.230201.0-dev1"}
```

Comment:
let's keep it YYYYMMDD
```suggestion
CLIENT_VERSION=${CLIENT_VERSION:-"1.20230201.0-dev1"}
```

Reply:
Agreed. Updated spec as well.